example_config/EUR-Lex-57k/bigru_lwan.yml

```diff
@@ -3,8 +3,10 @@ data_dir: data/EUR-Lex-57k
 data_name: EUR-Lex-57k
 min_vocab_freq: 1
 max_seq_length: 500
-include_test_labels: true # to compare Micro-F1 on zero-shots
-
+# We set include_test_labels to true for comparison with Chalkidis et al. (2019),
+# which handle zero-shots and report Micro-F1.
```

Comment:
we use "which" to refer to the paper. Then we should have "handles ... and reports ..." Or you can do "handled ... reported ..."

Reply:
Fixed, thanks!